widgetsnbextension/src/manager.js

```diff
@@ -567,10 +564,7 @@ WidgetManager.prototype._updateCellSnapshots = function(cell, index) {
         // Create a mime bundle.
         const bundle = {};
         bundle[imageMimetype] = imageData;
-        bundle['text/html'] = widgets.generateEmbedScript(
-            widgetState,
-            'Could not render widget, embed-manager missing from page context.'
-        );
+        bundle['text/html'] = widgets.generateEmbedScript(widgetState, imageData);
```

Comment:
This should be imageDataUrl , which is the b64 data + the mimetype required for the browser to know how to render it.